### PR TITLE
Add survey data

### PR DESCRIPTION
**Pull request by aays**
Not sure how much we'll be able to use this after all, but here it is all the same. I created two files: one with all survey data, and another with just quantitative answers. Addresses issue #12. 

I can also upload the script I used to make these as a separate file if needed - it's not super fancy:

```R
library(dplyr)
library(readr)

d1 <- read_csv('Desktop/Course-feedback-overall.csv') %>% 
  mutate(year = 2017)
d2 <- read_csv('Desktop/EEB313 Overall Course Feedback.csv') %>% 
  mutate(year = 2018)

surveys <- bind_rows(d1, d2) %>% 
  select(year, everything()) # reorder columns

surveys_quantitative <- surveys %>% 
  select_if(function(x) is.numeric(x))

write_csv(surveys, 'course-surveys.csv')
write_csv(surveys_quantitative, 'course-surveys-quantitative.csv')
```

### CODE REVIEW

**Comment by lwjohnst86** [comment]
We're going to put this on hold until we hear back from Ethics.

File: data/course-surveys.csv [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+2018,2018/12/07 3:09:42 AM EST,5,"If GitHub is going to be used, it is better to be taught earlier so by the time the projects started, they will have enough knowledge of Git.",2,5,GitHub assessment,4,not include GitHub,"The only difference is that we had now time to work in the project during class, something I really appreciate",4,5,5,"If GitHub is used, focus a little more on it, not just 1 lecture","Making graphs, as I did not use dplyr before, which was an easier method","R skills, statistical understanding.",No,Yes,5
+2018,2018/12/07 4:23:25 PM EST,5,"Challenges were great, helped make sure I was on track during the lecture.",3,5,Some instructors went a bit fast/taught more for an audience that was more familiar with programming/R in general and it would have been nice to slow down and do a bit more explaining.,5,no idea,"having ample time in class to work on the final project.
+
```
Comment (lwjohnst86) [inline data/course-surveys.csv]: There's a problem here. Seems the quote didn't work.

File: data/course-surveys.csv [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+2017,2017/12/05 3:00:44 PM EST,4,More challenges. Some of the GitHub stuff isn't on the online lectures which made it hard to remember what we learned.,3,4,Pacing,4,More in-depth outline of what is needed and more time to work on the project,"My group was easy to work with, the biggest difference is actually using R and GitHub",4,5,3,Make it a full year course and ask more ecological/biological questions during the assignment instead of just make this graph with a code we can copy off the lectures online,The feeling how actually learning something useful. I really like the finial project because it linked ecology and using R which I felt was lacking for the rest of the course.,Yes. I learned how to code and model in R which I've had fears about and how to use GitHub,no,Yes,5
```
Comment (lwjohnst86) [inline data/course-surveys.csv]: Do you mind stripping the time component of the stamp? And just keep the date?

File: data/course-surveys.csv [hunk 3]
```diff
@@ -0,0 +1,25 @@
...would you rate the course *lecture* material overall?,"How can the course material be improved (for example, the pacing, structure, content, concepts, and/or challenges)?",The ratio of instructors to students was:,How would you rate the instructors overall?,What can the instructors improve upon?,How would you rate the final project?,How could the final project be improved?,Which were the biggest differences between the final project in this class and your previous undergraduate group projects?,Did the lectures prepare you adequately for the final project?,How relevant do you think the concepts discussed during class will be for your future work/studies?,Did you need to spend more or less time and effort on this course than on other courses?,Is there anything specific we could do to help make this course better for future semesters?,What did you enjoy the most about the course and why?,"Were there any skills that you developed more in this course compared to your previous courses? If so, which ones?",Was there anything you had hoped/expected to learn starting out in this course that we did not cover?,Would you recommend this course to a friend?,How would you rate the course overall?
```
Comment (lwjohnst86) [inline data/course-surveys.csv]: Could you make the columns easier to read/use? (e.g. "Date", "OverallRating", "ImprovementsFeedback", etc)

File: data/course-surveys-quantitative.csv [hunk 4]
```diff
@@ -0,0 +1,23 @@
+year,How would you rate the course *lecture* material overall?,The ratio of instructors to students was:,How would you rate the instructors overall?,How would you rate the final project?,Did the lectures prepare you adequately for the final project?,How relevant do you think the concepts discussed during class will be for your future work/studies?,Did you need to spend more or less time and effort on this course than on other courses?,How would you rate the course overall?
```
Comment (lwjohnst86) [inline data/course-surveys-quantitative.csv]: Could you use better column headers?

**Comment by lwjohnst86** [comment]
Thanks @aays! Could you make those changes I requested, as well as include the R code as a file in the PR? Thanks!!!! :D 

Oh, and by the way, Ethics seems to be fine with using the information for research :)

**Comment by aays** [comment]
@lwjohnst86 Thanks for the review! The files have been updated and I've added the script as well. I also added the raw files in a separate commit just in case.

**Review by lwjohnst86 — APPROVED** [review]
Looks great!

**Pull request merged**